spec/models/user_spec.rb

```diff
@@ -160,4 +162,98 @@
       end
     end
   end
+
+  describe '#visible_assessments' do
+
+    context "when there are no assessments" do
+      let(:new_user) {create :user, user_name: 'testusername', first_name: 'testfname',
```

Comment:
If you don't care about username, last name, etc. for your tests. You can just do: 

```ruby
create :student
```

The factory will fill in values as needed.